### DISCUSSION TITLE
Support for multi column layout

### DISCUSSION THREAD

**Discussion by willopez** [yoopta-editor/Yoopta-Editor#59] [Ideas]
Hello Akhmed and congrats an you amazing feat of engineering.

Wondering if V2 will have layout components, for example I want to have two column row layout, and be able to add content within each column? Maybe this could be a custom component or a plugin?

**Comment by Darginec05** [comment]
Hi @willopez.

I always try to build on the needs of developers.
So, I'm not sure if it will be in version 2, but I will definitely add this cool feature to the next major version.

Thanks for idea!
